packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -0,0 +1,128 @@
+import * as core from '@aws-cdk/core';
+import * as cfn_type_to_l1_mapping from './cfn-type-to-l1-mapping';
+import * as futils from './file-utils';
+
+/**
+ * Construction properties of {@link CfnInclude}.
+ */
+export interface CfnIncludeProps {
+  /**
+   * Path to the template file.
+   *
+   * Currently, only JSON templates are supported.
+   */
+  readonly templateFile: string;
+
+  /**
+   * Whether the resources should have the same logical IDs in the resulting CDK template
+   * as they did in the original CloudFormation template file.
+   *
+   * **Note**: regardless of whether this option is true or false,
+   * the {@link CfnInclude.getResource} method always uses the original logical ID of the resource,
+   * as specified in the template file.
+   *
+   * @default true
+   */
+  // readonly preserveLogicalIds?: boolean;
+}
+
+/**
+ * A construct that allows you to import an existing CloudFormation template file into your CDK application.
+ * You can retrieve any of the resources defined in the template file by calling the {@link getResource} method.
+ * Any modifications you make on the returned resource objects will be reflected in the resulting CDK template.
```

Comment:
Same comment around language as above.

"Construct to import an existing CloudFormation template into the CDK application"

Similar to the language you have for `getResource()`.

Reply:
Done.